### ISSUE TITLE
Rules does not apply

### ISSUE THREAD

**Issue by IFtech-A** [U-C-S/Hurl#139]
## Summary

Clicking links on slack, telegram works on default browser policy(Last opened profile).

Tested with setting default browser to Hurl and without.

Testing ruleset match is working correctly.

## Actual Result

Open https://example.com on Chrome `Default` profile

## Expected result

Open https://example.com on Chrome `Profile 7` profile

## Environment

 - Browser : Chrome Version 126.0.6478.127 (Official Build) (64-bit)
 - App version: v0.8.3
 - OS: Windows
Config: 
```json
{
  "$schema": "https://raw.githubusercontent.com/U-C-S/Hurl/main/Utils/UserSettings.schema.json",
  "LastUpdated": "7/12/2024 11:06:44 AM",
  "Browsers": [
    {
      "Name": "Firefox",
      "ExePath": "C:\\Program Files\\Mozilla Firefox\\firefox.exe"
    },
    {
      "Name": "Google Chrome",
      "ExePath": "C:\\Program Files\\Google\\Chrome\\Application\\chrome.exe",
      "LaunchArgs": "--profile-directory='Profile 7'",
      "AlternateLaunches": [
        {
          "ItemName": "Example 1",
          "LaunchArgs": "--profile-directory='Profile 1'"
        },
        {
          "ItemName": "Example 2",
          "LaunchArgs": "--profile-directory=\"Profile 7\""
        }
      ]
    },
    {
      "Name": "Microsoft Edge",
      "ExePath": "C:\\Program Files (x86)\\Microsoft\\Edge\\Application\\msedge.exe"
    }
  ],
  "Rulesets": [
    {
      "Name": "Example",
      "Rules": [
        "r$https://example.com*"
      ],
      "BrowserName": "Google Chrome",
      "AltLaunchIndex": 1
    }
  ]
}
```

**Comment by U-C-S** [comment]
Seems already fixed in #117 , but it's still not released

Will create a new release `v0.8.4`, Thanks for reporting

**Comment by JackW-netizen** [comment]
Thanks for the fix. Rules working perfectly now.

**Comment by U-C-S** [comment]
Fix available in `v0.9.0-alpha-2` release
